cluster-scope/overlays/nerc-ocp-prod/certificates/default-ingress-certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: default-ingress-certificate
+  namespace: openshift-ingress
+spec:
+  issuerRef:
+    name: letsencrypt-staging-dns01
```

Comment:
Note that I have intentionally left this configured with the staging server for now; once cert generation has been verified to work correctly we'll swap these to the production issuers.